test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -229,7 +234,51 @@ private bool CompareDatabaseSchemas(string databaseName1, string databaseName2)
                         (d.TargetObject?.ObjectType.Name == i.type && d.TargetObject?.Name?.ToString() == i.name)))
                 .ToList();
 
-            return remainingDifferences.Count == 0;
+            // Some of the schema changes we are making to tables include addition of columns. In order to support
+            // upgrading older schemas to the newer ones we have to add default constraints to the upgrade script.
+            // These constraints will not be present in databases that were directly initialized with the latest schema.
+            // We need to exclude these constraints from the schema difference comparison.
```

Comment:
Is there a downside to just always having this as a default? Having schemas diverge or behave differently depending on how you got there seems a little scary

Reply:
I don't think there should be any downside to having the constraint added to the "fresh" schema (except that it is not actually required). In this situation, I don't think the schema difference would be an issue since the difference is just an additional default rule (and not, say, a different data type). 
I do like the idea of minimizing schema differences as much as possible to help keep things cleaner (especially when we are at v99 :)). I have updated the schema to add these constraints to 18.sql as well.